tests/core/test_IOManager.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import print_function
+import mock
+import unittest
+import os
+import threading
+import select
+
+#Make test version- agnostic:
+from sys import version_info
+if version_info.major == 2:
+  import __builtin__ as builtins    # pylint:disable=import-error
+else:
+  import builtins    # pylint:disable=import-error
+
+import IOManager
```

Comment:
The imports are clearer when we import the classes rather than the entire module.
thus
from IOManager import IOManager